plugins/woocommerce-blocks/assets/js/blocks/coming-soon/entire-site.scss

```diff
@@ -202,3 +202,28 @@ body:has(.woocommerce-coming-soon-banner) {
 		aspect-ratio: 1 / 1;
 	}
 }
+
+// Product image gallery template styles
+.woocommerce-coming-soon-image-gallery {
+	.wp-block-loginout {
+		display: flex;
+
+		a {
+			box-sizing: border-box;
+			border-radius: 6px;
+			gap: 10px;
+			font-size: 14px;
+			font-style: normal;
+			line-height: normal;
+			text-align: center;
+			text-decoration: none;
+			padding: 12px 16px;
+		}
+	}
+
+	.wp-block-image {
+		img {
+			border-radius: 14px;
```

Comment:
Good idea! I fixed this in 26d9b59. The sample images have a border embedded in the image file too. It might be worth a follow-up PR to remove this as well so users can adjust the radius on these?

Reply:
> The sample images have a border embedded in the image file too. It might be worth a follow-up PR to remove this as well so users can adjust the radius on these?

@adrianduffell Right, sounds good for a follow-up! Might need to convert the images to png to make it transparent 🤔

Reply:
On a second thought, I looked at the images when it's converted to png, the size increase is quite substantial. Given that this may only benefit small percentage of users who'd likely to reduce or remove the border radius, I think the better solution might be to square image without any border radius, so it will not have that odd white borders when users remove border radius.

Reply:
(the source images on figma have rectangular corners)

Reply:
Created a follow-up issue: https://github.com/woocommerce/woocommerce/issues/54036